docs/pages/develop/development-builds/introduction.mdx

```diff
@@ -1,88 +1,78 @@
 
-When you start a new project with Expo, the fastest and easiest way to get to "Hello, world!" is with [Expo Go](/get-started/set-up-your-environment/) on your device because you don't need to compile any native code or install any native tooling.
+All of these are native assets that get shipped with the native bundle and are immutable once installed. The Expo Go app does show a "splash screen which" which is just your app icon on a solid color background. This is a dev-only emulation of a splash screen we added so you could see what the splash screen will probably look like. It is limited however, e.g. you cannot test `SplashScreen.setOptions` to animate the splash screen.
 
-Beyond "Hello, world!" and prototypes, you'll quickly encounter limitations and need to build a development build of your app. For example, the Expo Go sandbox environment is limited to only the native packages included in the [Expo SDK](/versions/latest/), while you can include any library in a development build because it's just a normal native app.
+### Remote push notifications
 
-You can think of a development build as your fully customizable version of Expo Go, tailored specifically for your project. Or, conversely, Expo Go is a development environment with a preset runtime.
+While in-app [notifications](https://docs.expo.dev/versions/latest/sdk/notifications/) are available in Expo Go, remote push notifications (that is, sending a push notification from a server into the app) are not. This is because a push notification service should be tied to your own push notifications certificates and while it is possible to make it work in Expo Go, it often caused confusion for production builds. It is therefore better to test remote push notifications in Development Builds so you can ensure parity in behavior between development and production.
 
-</Collapsible>
+### Universal Links / App Links
 
-<Collapsible summary="What other types of builds are there?">
+Both [iOS Universal Links](https://docs.expo.dev/linking/ios-universal-links/) and [Android App Links](https://docs.expo.dev/linking/android-app-links/) require a two-way association between the native app and the website. In particular, it requires for the native app ro include the URL for the lined website. This is not possible with Expo Go due to the aforementioned native code immutability.
 
-- A [production build](/deploy/build-project/) is for the general public, deployed through stores.
-- A [preview build](/build/internal-distribution/) lets your team test your next release, either by installing an APK directly on Android or using ad hoc or enterprise provisioning on iOS.
+### (iOS Device only) open projects using older SDKs
 
-</Collapsible>
+Expo Go can only support one SDK version at a time. So, when we release a new SDK version we need to update Expo Go to support the new SDK version and this will be the only version of Expo Go available to install from the stores.
```

Comment:
```suggestion
Expo Go can only support one SDK version at a time. When a new SDK version is released, Expo Go is updated to support the newer version, and this will be the only version of Expo Go available to install from the stores.
```